packages/calcite-components/src/components/input-date-picker/input-date-picker.e2e.ts

```diff
@@ -351,6 +351,33 @@ describe("calcite-input-date-picker", () => {
       });
     });
 
+    it("closes when Escape key is pressed and focusTrapDisabled=true", async () => {
+      page = await newE2EPage();
```

Comment:
We’ll also need an issue to clean up this and similar tests. Avoiding `beforeEach` setups that get overwritten would help as they lead to discarded test pages.

Reply:
[Refactor tests to avoid beforeEach setups that get overwritten](https://github.com/Esri/calcite-design-system/issues/10892#top) #10892